android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -90,8 +90,19 @@ private Boolean isEmulator() {
   }
 
   private Boolean isTablet() {
-    int layout = getReactApplicationContext().getResources().getConfiguration().screenLayout & Configuration.SCREENLAYOUT_SIZE_MASK;
-    return layout == Configuration.SCREENLAYOUT_SIZE_LARGE || layout == Configuration.SCREENLAYOUT_SIZE_XLARGE;
+    boolean layout = (getReactApplicationContext().getResources().getConfiguration().screenLayout & Configuration.SCREENLAYOUT_SIZE_MASK) >= Configuration.SCREENLAYOUT_SIZE_LARGE;
+    if (layout) {
+      DisplayMetrics metrics = new DisplayMetrics();
```

Comment:
I got a build error on this line: `error: cannot find symbol DisplayMetrics`. Did you forget to import android.util.DisplayMetrics?

Reply:
Nice catch!

Reply:
Yes fixing.